src/components/ArticleById.jsx

```diff
@@ -0,0 +1,29 @@
+import { useEffect, useState } from "react";
+import { getArticleById } from "../api";
+import ArticleCard from "./ArticleCard";
+
+export default function ArticleById({ article_id }) {
+  const [article, setArticle] = useState({});
+
+  useEffect(() => {
+    getArticleById(article_id).then((data) => {
+      //console.log(data, "art in comp");
+      setArticle(data.article);
+    });
+  }, []);
+  return (
+    <main>
+      <ArticleCard
+        article_id={article.article_id}
+        title={article.title}
+        article_img_url={article.article_img_url}
+        topic={article.topic}
+        author={article.author}
+        created_at={article.created_at}
+        votes={article.votes}
+        comment_count={article.comment_count}
+        body={article.body}
+      />
+    </main>
```

Comment:
The text is currently unreadable as it's white on a near-white background, so the styling will need updating to make it accessible please 🎨 
<img width="560" alt="image" src="https://github.com/jgroma/fe-nc-news/assets/9512932/450526ba-e234-445e-bee1-857ba93126e7">